scale: 0.04
relative: 20.0
Q: [0.0225,0.0225,1.0,0.125,0.125]
R: [0.0013,0.0007,0.1]
# new_value = old_value + Kalman_gain * (detection - old_value)
one_d_kalman_gains: [0.62, 0.5, 0.5, 0.75, 0.3]   # Kw, Kl, Kh, Kyaw, Kconfidence
confidence_drop: 1.2    # How much should confidence drop over time c(t) = c(0) * exp(-1 * confidence_drop * t)
metricGate: 0.5
delete_time: 0.5        # How many seconds to wait before deleting an object with no associated detections
metric_thres: 0.4       # metric distance
unknown_type: 3         # This type corresponds to an "unknown" object
# Scaling factors for cost function parameters: d = d_euclid + alpha * d_size + beta * d_confidence
cost_alpha: 0.25
cost_beta: 0.5
# HiddenMarkovModel Parameters
types: [1, 3]           # [Pedestrian, Unknown]
# p(z_t = i | z_{t-1} = j)
A_hmm: [0.99, 0.28,        # Transition Matrix (Columns must sum to one)
        0.01, 0.82]
# p(x_t = l | z_t = k)
C_hmm: [0.90, 0.32,       # Observation Matrix (Columns must sum to one)
        0.10, 0.68]
# initial p(z_0) guess
pi_hmm: [0.5, 0.5]
